### PR TITLE
(feat) (auth-portal): Remove tutorial and fix wording

### PR DESCRIPTION
**Pull request by keeb**
The tutorial has become out of date and was last updated 5 months ago. It has served its purpose and will be re-implemented once the SaaS is in place.

For now, we are recommending users either use a dev environment or reach out to us for a temporary vm.

### CODE REVIEW

File: app/auth-portal/src/content/tutorial/02-install_and_run_system_initiative.md [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
-title: Install and run System Initiative
+title: Run System Initiative
 hideWorkspaceLink: true
 ---
 ## Install and run System Initiative
 
-Let's do this:
-* Visit the <router-link :to="{ name: 'download' }" target="_blank">Install page</router-link> and follow the instructions to install System Initiative.
-* Run the command `si start` in a terminal window, which will check for system dependencies, prompt for the necessary credentials, and download/start the System Initiative components.
+We're currently working on implementing a SaaS version of System Initiative that will alleviate the need install any software.
```
Comment (nickgerace) [inline app/auth-portal/src/content/tutorial/02-install_and_run_system_initiative.md:7]:
```suggestion
We're currently working on implementing a SaaS version of System Initiative that will alleviate the need to install any software. 
```

File: app/auth-portal/src/content/tutorial/02-install_and_run_system_initiative.md [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
 
-The button below should have two red beacons, which will turn green and say "Frontend online" and "Backend online" when you're running System Initiative and ready to launch. Click this button to login and open your workspace when it's ready:
+In the meantime, there's 2 alternatives: 
+
+* You can build from source, using buck2 and tilt. It takes a little bit to get started but the end result is a full development environment that mirrors what the team is doing and is at the moment the most stable way to use System Initiative.
+
+* We can invite you to a private workspace as a playground. This is ephemeral and should be considered as such. This environment would be used to validate one of your ideas or to get some hands on experience with System Initiative. 
+
+
+In both cases, please [join the Discord](https://discord.com/invite/system-init) and reach out to @SI_keeb for any help or assistance.
```
Comment (nickgerace) [inline app/auth-portal/src/content/tutorial/02-install_and_run_system_initiative.md:16]: I'm okay with one person being explicitly named, but just want to double check... we want this, right?

**Review by nickgerace — APPROVED** [review]
Looks great! The `README` (and maybe other dev docs) will need to be updated as well, but we can do that later.

File: app/auth-portal/src/content/tutorial/02-install_and_run_system_initiative.md (hunk 2 above)
Comment (keeb) [reply app/auth-portal/src/content/tutorial/02-install_and_run_system_initiative.md:16]: It is just a few weeks until all of this goes away so it's just a stop gap. I am fine with it.

Comment (theoephraim) [reply app/auth-portal/src/content/tutorial/02-install_and_run_system_initiative.md:16]: I think "hands-on" is usually hyphenated?

**Comment by theoephraim** [comment]
IMO we also need to also remove the download page, links to it in the header, and the "Install" link on the corp site home page. I've always found it confusing, but now it would be even more confusing...

**Comment by keeb** [comment]
@theoephraim I'll work on adding that feedback to this PR

**Comment by keeb** [comment]
bors merge

**Pull request merged**